aidermacs.el

```diff
@@ -595,7 +596,14 @@ The full command will be \"COMMAND-PREFIX <current buffer file full path>\"."
 (defun aidermacs-drop-current-file ()
   "Drop the current file from aidermacs session."
   (interactive)
-  (aidermacs-act-on-current-file "/drop"))
+  (if (not buffer-file-name)
```

Comment:
`aidermacs-act-on-current-file` will not be used anymore after your change, please remove the function.

Reply:
This actually got clobbered in the rebase. `aidermacs-act-on-current-file` is updated to work with tramp, so I could keep the file and refactor this to use that function, or I can delete it as you suggest. Any preference?

Reply:
Let's delete it, `aidermacs-act-on-current-file` was a leftover hanging fruit from a previous implementation anyways,

Reply:
Deleted 👍